tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml

```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
-		control information to map a TSN Stream to a DetNet flow. 
-		N:1 mapping (aggregating TSN Streams in a single
-		DetNet flow) SHALL be supported. The management or control
-		function that provisions flow mapping SHALL ensure that
-		adequate resources are allocated and configured to provide
-		proper service requirements of the mapped flows.
-	</t>
-	<t>
-		Due to the (intentional) similarities of the DetNet PREOF and TSN FRER 
-		functions service protection function interworking is possible between 
-		the TSN and the DetNet domains. Such service protection interworking 
-		scenarios MAY require to copy sequence number fields from TSN (L2) to 
-		PW (MPLS) encapsulation. However, such interworking is out-of-scope in 
-		this document and left for further study.
-	</t>
+                Implementations of this document SHALL use management and
+                control information to map a TSN Stream to a DetNet flow. 
+                N:1 mapping (aggregating TSN Streams in a single
+                DetNet flow) SHALL be supported. The management or control
+                function that provisions flow mapping SHALL ensure that
+                adequate resources are allocated and configured to provide
+                proper service requirements of the mapped flows. -->
+        </t>
+        <t>
+                Due to the (intentional) similarities of the DetNet PREOF and
+                TSN FRER functions service protection function interworking is
+                possible between the TSN and the DetNet domains. Such service
+                protection interworking scenarios MAY require to copy sequence
+                number fields from TSN (L2) to PW (MPLS) encapsulation.
+                However, such interworking is out-of-scope in this document and
+                left for further study.
+        </t>
+        <t>
+                The Service proxy SHALL support TSN MAC addresses allowing a
+                TSN connection to connect to another TSN network over the
+                DetNet Network. A service proxy SHALL be configured as either a
+                unicast DetNet application or a multicast application.  Service
+                proxies that support unicast service SHALL support TSN unicast
+                MAC addresses and Service proxies that support multicast
+                services SHALL support TSN multicast addresses.  The TSN
+                specific functions on the DetNet Edge Node SHOULD participate
+                in TSN Stream reservation protocols allowing TSN to establish
+                flows to the DetNet Edge node.  At the DetNet Edge node the TSN
+                functions are tunneled through the DetNet flow.
+        </t>
+        <t>
+                A static MPLS DetNet flow is configured to carry any number of
+                TSN flows. The Service proxy bandwidth profile SHOULD match the
+                bandwidth allocated to the MPLS Detnet Flow.  
+        </t>

```

Comment:
What is a "static MPLS Detnet flow"? I think You mean here "MPLS DetNet flow".
I think the service proxy does not have a BW profile. Service proxy is a function. The DetNet service definition has a BW profile.